web_console/customizing-the-web-console.adoc

```diff
@@ -17,6 +17,11 @@ include::modules/creating-custom-links.adoc[leveloffset=+1]
 
 include::modules/customizing-the-web-console-URL.adoc[leveloffset=+1]
 
+[NOTE]
+====
+When a deployment reaches its preset limits or quotas, a message appears in the *Topology* and *List* views of the web console.
```

Comment:
Per Andrea's comment, I'd also address _where_ the "Topology" and "List" views are, e.g:

"When a deployment reaches its preset limits or quotas, a message appears in the **Topology** view of the **Developer** perspective, or in the **List** view of X in the web console."

I don't know where the List view is or what this is referring to.

Reply:
did not make an admonition because of text expansion.